docs/sources/administration/provisioning.md

```diff
@@ -15,7 +15,7 @@ In previous versions of Grafana, you could only use the API for provisioning dat
 
 ## Config File
 
-Checkout the [configuration](/installation/configuration) page for more information on what you can configure in `grafana.ini`
+Checkout the [configuration]({{< relref "../installation/configuration" >}}) page for more information on what you can configure in `grafana.ini`
```

Comment:
This relative link looks weird - is the `../` needed? A lot more of those in the changes.

Reply:
Unfortunately yes. In the website, these pages will be several directories deep, so referencing a page in another directory requires moving up a level.